src/systemtask/SystemTask.cpp

```diff
@@ -102,7 +102,9 @@ SystemTask::SystemTask(Drivers::SpiMaster& spi,
                      spiNorFlash,
                      heartRateController,
                      motionController,
-                     fs) {
+                     fs),
+    wokenBy {WokenBy::Other},
```

Comment:
please move the default values to the definition in the header, just my personal preference to keep the constructor for non-default values

Reply:
done